mirage/apt/apt_inputs.py

```diff
@@ -677,14 +612,19 @@ def get_pointing_info(self, file, propid):
                             type_str.append(elements[18])
                             expar.append(np.int(elements[19]))
                             dkpar.append(np.int(elements[20]))
-                            ddist.append(np.float(elements[21]))
+                            if elements[18] == 'PARALLEL':
+                                ddist.append(None)
+                            else:
+                                ddist.append(np.float(elements[21]))
                             # For the moment we assume that the instrument being simulated is not being
                             # run in parallel, so the parallel proposal number will be all zeros,
                             # as seen in the line below.
                             observation_id.append("V{}P{}{}{}{}".format(vid, '00000000', vgrp, seq, act))
```

Comment:
I think that's correct as is. The yaml filename follows the just naming convention, which is jw**yaml. But the observation ID should be in the (e.g) V80600013001P0000000002101 format. See the keyword dictionary entry for OBS_ID: https://mast.stsci.edu/portal/Mashup/Clients/jwkeywords/index.html

Reply:
Oh! I see. 
In that case, it might be useful to include a new `'filename'` key in the pointing dictionary, so we have a connection between the dictionary and the files.
I see that `yaml_generator` adds a `yamlfile` key, but that isn't actually in the same format as the yaml files (e.g. Act01_A3_imaging_Dither01.yaml).

Reply:
I suggest we take this into a separate PR. I have all the necessary information from a Help Desk ticket I opened.

Reply:
See #205